topics/tutorials/file_and_code_templates/using_file_templates.md

```diff
@@ -0,0 +1,131 @@
+[//]: # (title: Using File Templates Programmatically)
+
+<!-- Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+File templates provided by a plugin can be used during new file creation, in code intention actions, or other plugin features.
+They can be accessed with the
+[`FileTemplateManager`](upsource:///platform/lang-impl/src/com/intellij/ide/fileTemplates/FileTemplateManager.java)
+service providing methods returning all or single file templates from a given category.
+For example, to obtain a template from the _[Code](providing_file_templates.md#code)_ category, pass its name to the `getCodeTemplate()` method (notice the lack of the `.ft` extension):
```

Comment:
`<path>.ft</path>`

Reply:
As in one of the previous comments, please double confirm.

Reply:
Thanks, fixed.